### ISSUE TITLE
Can we move PixelTracks used for HI reco to patatrack PixelTracks, using GPU?

### ISSUE THREAD

**Issue by clacaputo** [cms-sw/cmssw#37425]
This issue is intended to initiate a discussion about whether the pixeltracks used for HI reco can be moved to patatrack pixel tracks that also run on GPU. 

This would be an advantage also for 2022 data taking since the HI reco happens during the Christmas period at CERN also using the HLT farm that has gpus.

**Comment by clacaputo** [comment]
assign reconstruction
assign heterogeneous

**Comment by clacaputo** [comment]
Tagging HI reco contact: @mandrenguyen

**Comment by mandrenguyen** [comment]
To give some context, we have been running stand-alone pixel tracking in the offline heavy-ion reconstruction for a while.
We recently moved to the fit model used by patatrack #36558
We are running the patatrack code online, as is done in pp.  I understand that this code is visible offline, as well.
I don't know of any good reason not to simply run the patatrack code. 
 I propose to organize a chat to discuss this. 
 On the heavy-ion side the tracking is manged by @CesarBernardes and @abaty 
I've already been contacted by @AdrianoDee from the patatrack team.
I'll put the reco conveners in the loop, and we can add in any other relevant people from there.

**Comment by VinInn** [comment]
I suggest you just try and eventually report issues.

**Comment by jpata** [comment]
type trk

**Comment by jpata** [comment]
@mandrenguyen could you summarize the conclusions of the meeting? Is it planned to be implemented by HI, or should we close this issue?

**Comment by mmusich** [comment]
type tracking

**Comment by mmusich** [comment]
@AdrianoDee @cms-sw/reconstruction-l2 is this issue still active after the merge of https://github.com/cms-sw/cmssw/pull/41632 ?

**Comment by AdrianoDee** [comment]
Could be closed indeed.

**Comment by iarspider** [comment]
@cmsbuild please close
